src/game/boe.actions.cpp

```diff
@@ -1402,6 +1473,30 @@ void update_item_stats_area(bool& need_reprint) {
 	need_reprint = true;
 }
 
+location mouse_window_coords() {
+	location where_curs = sf::Mouse::getPosition(mainPtr());
+	where_curs = mainPtr().mapPixelToCoords(where_curs, mainView);
```

Comment:
Did you test this on Mac? It looks like the kind of code that might need the "input monitoring" permission, but I'm not entirely sure. Maybe it's more lenient with mouse than keyboard.

Reply:
Yeah, I wrote it on mac, but I did have the input permissions enabled. So just now I turned off the input permissions to see if it still worked, and it does, so I think it should be fine?